installscripts/wizard/scenarios/stack_with_dockerized_gitlab_jenkins.py

```diff
@@ -35,6 +37,8 @@ def start(parameter_list):
         print("Deploying Dockerized SonarQube server==============>")
         get_and_add_docker_sonar_config(get_docker_path() + "/sonar/")
 
+    get_and_add_splunk_config(get_terraform_folder())
```

Comment:
So this is only an option for scenario 3?

Reply:
No, here also https://github.com/tmobile/jazz-installer/pull/361/files#diff-029cd61ecdc2eb0d78932fbf9958912b